tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -356,6 +356,23 @@ Fetch Image Tooltip Info
     Click Element  //div[@class='jsp-app__header__title']
     [Return]  ${tmp_list}
 
+Handle Bad Gateway Page
+    [Documentation]    It reloads the JH page until Bad Gateway error page
+    ...                disappears. It is possible to control how many
+    ...                times to try refreshing using 'retries' argument
+    [Arguments]   ${retries}=10
+    Capture Page Screenshot    jh_badgateway_kw.png
+    FOR    ${counter}    IN RANGE    0    ${retries}+1
+        ${bg_present} =    Run Keyword And Return Status    Page Should Contain    Bad Gateway
+        IF    $bg_present == True
+            Reload Page
```

Comment:
Maybe you could Sleep a second or two before reloading the page

Reply:
+1, I guess that without sleeps the 10 retries will happen in less than 10 seconds, it might not be enough time for JH to recover

Reply:
the jenkins report is not available anymore, so i don't remember the time it takes, but yes I can set a sleep interval using an argument and setting as default 1 sec

Reply:
done